src/views/tutorial-view/components/image-config/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+import ImageConfig from '.'
+
+const testImage = (
+  <img alt="hashicorp logo" src="/img/favicons/favicon-192x192.png" />
+)
+
+/**
+ * this is an approximation of this component's markup when used within an MDX document, eg:
+ * <ImageConfig>
+ *
+ *   ![hashicorp logo](/img/favicons/favicon-192x192.png)
+ *
+ * </ImageConfig>
+ */
+const renderBasicMock = () =>
```

Comment:
These tests were ported over from learn - [see original component](https://github.com/hashicorp/learn/tree/master/components/image-config). I've made modifications to ensure tests are still working as expected with the revised component.